rpaas/k8s.go

```diff
@@ -507,6 +508,146 @@ func (m *k8sRpaasManager) UnbindApp(ctx context.Context, instanceName string) er
 	return m.cli.Update(ctx, instance)
 }
 
+func (m *k8sRpaasManager) DeleteRoute(ctx context.Context, instanceName, path string) error {
+	instance, err := m.GetInstance(ctx, instanceName)
+	if err != nil {
+		return err
+	}
+
+	index, found := hasPath(*instance, path)
+	if !found {
+		return &NotFoundError{Msg: "path does not exist"}
+	}
+
+	var newLocations []v1alpha1.Location
+	for i, location := range instance.Spec.Locations {
+		if i == index {
```

Comment:
I'm not sure, but since the index variable can only have a single value in this function call, can't you use that colon syntax here?

```golang
newLocations := append(instance.Spec.Locations[:index], instance.Spec.Locations[index+1:]...) 
```

Reply:
It really works... I'm changing this guy. Thanks.